script/Arcueid/Day 7/07_00_ARC07_1_1.txt

```diff
@@ -36,7 +36,7 @@ I use the card key and pass through the door next to the gate.
 [sha:ee06a3a3c702bc1cc62484e0ff9dbcebace8c71b]{
 -- Page 132, Offset 16903.
 -- 玄関口に立ったところで、今朝のやりとりを思い出した。
-The moment I arrived at the entrance, I remembered the back and forth from this morning.
+The moment I arrive at the entrance, I remember the back and forth from this morning.
```

Comment:
What are you basing these tense changes off?

Reply:
The sequence and when the action occurred, like here for example he got through the gate, made an "Ah" sound, and then this line came, I don't think it's wrong to be in past tense, but such cases have usually been in present in the previous files.

Reply:
yeah it's a nasu mess. source text here is past tense :upside_down_face: